service/src/test/java/bio/terra/externalcreds/services/OAuth2ServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package bio.terra.externalcreds.services;
+
+import bio.terra.common.logging.LoggingInitializer;
+import bio.terra.externalcreds.config.ProviderConfig;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.security.oauth2.client.registration.ClientRegistration;
+import org.springframework.security.oauth2.core.endpoint.OAuth2AccessTokenResponse;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.stereotype.Component;
+
+@SpringBootApplication(scanBasePackages = {"bio.terra.externalcreds", "bio.terra.common.logging"})
+@Component
+public class OAuth2ServiceTest {
```

Comment:
I'd love to come up with a different way to do this, because as it is now someone has to manually run it separately; at the very least I'd like to have it somewhere different in the folder structure. I'll see if I can come up with a proposal.

Reply:
I do not expect to have to run this often. This is just to have something so that when people fiddle with this code they have something easy to to use for quick iterations.